app/views/schools/add_participants/cannot_find_their_details.html.erb

```diff
@@ -5,42 +5,44 @@
 <% content_for :before_content, govuk_back_link( text: "Back", href: back_link_path) %>
 
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
-    <span class="govuk-caption-xl"><%= @school.name %></span>
-    <h1 class="govuk-heading-xl">We cannot find <%= add_participant_form.full_name.titleize %>’s record</h1>
-    <p>Check the information you’ve entered is correct.</p>
-    <p>We need to find <%= add_participant_form.full_name.titleize %> in the Teaching Regulation Agency records to make sure they’re eligible for this funded programme.</p>
-
-    <%= govuk_summary_list do |summary_list| %>
-      <% summary_list.row do |row| %>
-        <% row.key { "Name" } %>
-        <% row.value { add_participant_form.full_name.titleize } %>
-        <% row.action(text: "Change",
-                      visually_hidden_text: "name",
-                      href: url_for( { step: :name } )) %>
-      <% end %>
-
-      <% summary_list.row do |row| %>
-        <% row.key { "TRN" } %>
-        <% row.value { add_participant_form.trn } %>
-        <% row.action(text: "Change",
-                      visually_hidden_text: "TRN",
-                      href: url_for({ step: :trn })) %>
-      <% end %>
-
-      <% summary_list.row do |row| %>
-        <% row.key { "Date of birth" } %>
-        <% row.value { add_participant_form.date_of_birth.to_date.to_s(:govuk) } %>
-        <% row.action(text: "Change",
-                      visually_hidden_text: "dob",
-                      href: url_for({ step: :dob })) %>
-      <% end %>
-    <% end %>
-
-    <h3 class="govuk-heading-s">If this information is correct.</h3>
-    <p>You can still continue to register <%= add_participant_form.full_name.titleize %>. We'll contact them to check their information.</p>
-
-    <%= govuk_button_link_to "Continue and confirm", step_schools_add_participants_path(step: :email)  %>
-
-  </div>
-</div>
+    <div class="govuk-grid-column-two-thirds">
```

Comment:
nit: most of the files here need a trailing space

Reply:
Done - and pushed up 
nit: corrected